packages/messaging-api-messenger/src/MessengerClient.ts

```diff
@@ -426,6 +426,49 @@ export default class MessengerClient {
       .then(res => res.data, handleError);
   }
 
+  /**
+   * User Level Menu
+   *
+   * https://developers.facebook.com/docs/messenger-platform/send-messages/persistent-menu#user_level_menu
+   */
+  getUserLevelMenu(
+    userId: string,
+    { accessToken: customAccessToken }: Types.AccessTokenOptions = {}
+  ): Promise<Types.UserPersistentMenu[]> {
+    return this._axios
+      .get(
+        `/me/custom_user_settings?psid=${userId}
+        &access_token=${customAccessToken || this._accessToken}`
+      )
+      .then(res => res.data.data, handleError);
+  }
+
+  setUserLevelMenu(
+    profile: Types.MessengerProfile,
+    { accessToken: customAccessToken }: Types.AccessTokenOptions = {}
+  ): Promise<Types.MutationSuccessResponse> {
+    return this._axios
+      .post(
+        `/me/custom_user_settings?access_token=${customAccessToken ||
+          this._accessToken}`,
+        profile
+      )
+      .then(res => res.data, handleError);
+  }
+
+  deleteUserLevelMenu(
```

Comment:
This should be `deleteCustomUserSetting(userId: string, params: ['persistent_menu'])`.